vulnerability-disclosure-policy.md

```diff
@@ -45,7 +45,7 @@ A subset of these systems may be eligible for bounties. Check our [program page
 **The following test types are not authorized:**
 
 * User interface bugs or typos.
-* Network denial of service (DoS or DDoS) tests.
+* Volume-based [denial of service (DoS or DDoS)](https://en.wikipedia.org/wiki/Denial-of-service_attack) tests.
```

Comment:
Been thinking on how to clarify. I think when we say "DoS tests" are excluded, it's being interpreted as DoS vulnerabilities are excluded, but that is not true, a vulnerability affecting Availability is certainly in scope. Does this sound like it effectively makes the distinction between the _tests_ being excluded, but not DoS vulnerabilities.

```suggestion
* Network [denial of service (DoS or DDoS)](https://en.wikipedia.org/wiki/Denial-of-service_attack) tests. This does not exclude vulnerabilities impacting Availability.
```